README.rst

```diff
@@ -0,0 +1,153 @@
+		http://localhost:18000/eox-core/api/v1/enrollment/ --header "Content-Type: application/json" \
+		--data '{"course_id": "course-v1:edX+DemoX+Demo_Course", "email": "edx@example.com",
+		"mode": "audit", "force": 1}'
+
+Installation on Devstack
+========================
+- Create an openedx Ironwood instace if it doesn't exist already:
+
+.. code-block:: bash
+
+	# Assuming ubuntu (if not replace apt-get for the package manager of your linux dist, e.g. yum)
+	sudo apt-get install direnv
+	cd ~/Documents/
+	mkdir eoxstack
+	cd eoxstack
+	git clone git@github.com:edx/devstack.git
+	cd devstack
+	git checkout open-release/ironwood.master
+	export OPENEDX_RELEASE=ironwood.master
+	make dev.checkout
+	make dev.clone
+	make dev.provision
+	echo export OPENEDX_RELEASE=ironwood.master > .envrc
+
+
+- Clone the git repo:
+
+.. code-block:: bash
+
+	cd ~/Documents/eoxstack/src/  # Change path as needed
+	sudo mkdir edxapp
+	cd edxapp
+	git clone git@github.com:eduNEXT/eox-core.git
+
+- Install plugin from your server (in this case the devstack docker lms shell):
+
+.. code-block:: bash
+
+	cd ~/Documents/eoxstack/devstack  # Change for your devstack path (if you are using devstack)
+	make lms-shell  # Enter the devstack machine (or server where lms process lives)
+	sudo su edxapp -s /bin/bash  # if you are using devstack you need to use edxapp user
+	source /edx/app/edxapp/venvs/edxapp/bin/activate
+	cd /edx/src/edxapp/eox-core
+	pip install -e .
+
+Development
+===========
+
+To update dependencies you must edit the .in files (not requirements.txt directly) and compile them using pip-tools:
+
+.. code-block:: bash
+
+	$ source /path/to/venv/bin/activate
+	(venv)$ pip install pip-tools
+	(venv)$ pip-compile
+
+Integrations with third party services
+======================================
+
+The plugin offers some integrations listed below:
+
+1. **Sentry**: this service allows to track the errors generated on edx-platform. Check more details in https://sentry.io/welcome/. To enable the integration, follow the steps below:
+
+	* Install the plugin with Sentry support (extras_require [sentry])
+
+	* Sign up/in to your sentry account and create a new Django application integration.
+
+	* Get the DSN for your integration. This is an unique identifier for your application.
+
+	* Setup the following configuration values for edx\-platform:
+
+		.. code-block:: yaml
+
+			EOX_CORE_SENTRY_INTEGRATION_DSN: <your DSN value>
+			EOX_CORE_SENTRY_IGNORED_ERRORS: [] # optional
+
+	By default, **EOX_CORE_SENTRY_INTEGRATION_DSN** setting is None, which disables the sentry integration.
+
+	**EOX_CORE_SENTRY_IGNORED_ERRORS** is optional. It is a list of exceptions that wants to be ignored by sentry. For instance, it can be defined as:
+
+		.. code-block:: yaml
+
+			EOX_CORE_SENTRY_IGNORED_ERRORS: [
+				'xmodule.exceptions.NotFoundError',
+				'openedx.core.djangoapps.user_authn.exceptions.AuthFailedError',
+			]
+
+Course Management automation compilation
```

Comment:
Period at the end of the line. @morenol

Reply:
@morenol  We can add a description like:

This component allows Studio users to make changes in multiple courses, such as:
- Add or remove staff/instructor users across multiple courses in one organization.
- Change course settings in multiple courses at once.
- Re-run a course across multiple organizations.

Reply:
I think that the period is not needed here since this is a title.

On the other hand, is that description related to `Course Management automation compilation`? @Squirrel18

Reply:
Let's rename this section to: Course Management automation and then create a new subsection containing the compilation instructions. @morenol